examples/geojson-vt.js

```diff
@@ -72,16 +72,21 @@ fetch(url)
       extent: 4096,
       debug: 1,
     });
-    const vectorSource = new VectorTileSource({
-      format: new GeoJSON({
-        // Data returned from geojson-vt is in tile pixel units
-        dataProjection: new Projection({
-          code: 'TILE_PIXELS',
-          units: 'tile-pixels',
-          extent: [0, 0, 4096, 4096],
-        }),
+    const format = new GeoJSON({
+      // Data returned from geojson-vt is in tile pixel units
+      dataProjection: new Projection({
+        code: 'TILE_PIXELS',
+        units: 'tile-pixels',
+        extent: [0, 0, 4096, 4096],
       }),
+    });
+    const vectorSource = new VectorTileSource({
       tileUrlFunction: function (tileCoord) {
+        // Use the tile coordinate as a pseudo URL for caching purposes
+        return JSON.stringify(tileCoord);
```

Comment:
```suggestion
        return tileCoord.toString();
```
would be a bit lighter. Feel free to commit or ignore.